numpy/random/_generator.pyx

```diff
@@ -3788,6 +3788,12 @@ cdef class Generator:
             Each entry ``out[i,j,...,:]`` is a ``p``-dimensional value drawn
             from the distribution.
 
+        Notes
+        -----
+
+        .. versionchanged:: 1.22.0
+            Added support for broadcasting of arguments
+
```

Comment:
I assume this refers to broadcasting the  `pvals`  against `n`. If so,  it would be better to put this after the out description and be specific, i.e.,  "\`pvals\`  against \`n\`" instead of "arguments". If not, it would be nice to know what is does refer to.

Reply:
Thanks, I just updated the language. Based on the docstring and changes from #16740, you're right that `pvals` are being broadcast to `n`.